i18n/zh-cn/docusaurus-plugin-content-docs/current/docs/references/users/README.md

```diff
@@ -166,4 +166,46 @@ Logto 默认使用 [Argon2](https://en.wikipedia.org/wiki/Argon2) 的实现 [nod
 
 ### 自定义数据（custom data）
 
+_custom data_ 即自定义数据，可以在这个可自定义的属性中保存额外的用户信息。
```

Comment:
if it's a field, should we use `custom_data`?
if it's a word, should we use _Custom data_ or _Custom Data_?

Reply:
It's a field. 

Use _custom data_ is enough for the developer to understand.

Using the code segment `custom_data` everywhere in the docs is not pretty, not neat, and unnecessary.

Need to intended to align this style with related content.

@gao-sun any idea?

Comparison:

![image](https://user-images.githubusercontent.com/10594507/177047367-1f63ab1b-8100-48aa-ba0b-b9d168252e19.png)

![image](https://user-images.githubusercontent.com/10594507/177047343-8b374dcd-eb87-4dd2-b9f3-1d8c27f37944.png)

Reply:
then we can use Custom Data

Reply:
updated. PTAL